cmd/frontend/graphqlbackend/search.go

```diff
@@ -96,6 +96,17 @@ func NewSearchImplementer(args *SearchArgs) (SearchImplementer, error) {
 		if err != nil {
 			return alertForQuery(args.Query, err), nil
 		}
+		// Get settings to check if CaseSensitiveSearch is active or not
+		ctx := context.Background()
+		settings, err := decodedViewerFinalSettings(ctx)
```

Comment:
```suggestion
		settings, err := decodedViewerFinalSettings(context.Background())
```
Nit: just inline it.